.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
+  "parserOptions": {
```

Comment:
A system test to check for ES5 as the specified version has been added, among other related checks.